api/server.go

```diff
@@ -42,6 +43,7 @@ func injectProfilerHandler(mux *http.ServeMux, profilingEnabled bool) {
 	}
 
 	mux.Handle("/debug/pprof/", handler)
+	mux.Handle("/metrics", promhttp.Handler())
```

Comment:
I have reused the `--profiling-enabled` flag, but I'm also happy with any other, separate, suggestion.

My rationale is that the potential user or use case for both probably overlaps most of the cases, like trying to understand how k6 behaves in terms of resource utilization, etc.